tools/exporter/exporter.py

```diff
@@ -470,6 +469,12 @@ def sync_exports(logger, sc_client, settings):
                     export_doc = sc_client.get_export(audit_id, timezone, export_profile_id, export_format)
                 elif export_format == 'json':
                     export_doc = json.dumps(audit_json, indent=4)
+                elif export_format == 'csv':
+                    csv_exporter = csv.CsvExporter(audit_json)
+                    export_filename = audit_json['template_id']
+                    csv_exporter.write(export_path, export_filename, 'ab')
+                    # skip call to save_exported_document, csv_exporter class handles it with call to write()
```

Comment:
Let's make this consistent and have a function that returns the export_doc here.